app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleSearch/SearchResults.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="flex justify-end gap-1 py-4 bg-white dark:bg-slate-900">
+    <div class="flex flex-col gap-1 w-full">
+      <div v-if="isLoading" class="empty-state-message">
+        <spinner size="" />
```

Comment:
Can we remove the size prop if its not needed?
```suggestion
        <spinner />
```